components/sections/Projects/ProjectTitle.jsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+const ProjectTitle = ({ name, openProjectPage, width, active, index }) => {
+
+  return (
+    <div
+      className="cursor-pointer relative overflow-hidden  group flex items-start"
+      onClick={width > 1024 ? openProjectPage: () =>{}}
+    >
+      <div className="text-13 leading-16 sm:text-16 sm:leading-18">{name}</div>
+      <div className={'transition-opacity duration-300 ' + (active === index ? 'opacity-100': 'opacity-0')}>
```

Comment:
I think if you are using double quotes, it's good to stay consistent throughout. You should change single quotes into double quotes to keep consistency.